src/lib.rs

```diff
@@ -517,35 +534,6 @@ impl Instance {
             id: wgn::wgpu_instance_get_gl_surface(self.id),
         }
     }
-
-    #[cfg(not(feature = "gl"))]
-    pub fn create_surface_from_xlib(
-        &self,
-        display: *mut *const std::ffi::c_void,
-        window: u64,
-    ) -> Surface {
-        Surface {
-            id: wgn::wgpu_instance_create_surface_from_xlib(self.id, display, window),
-        }
-    }
-
-    #[cfg(not(feature = "gl"))]
-    pub fn create_surface_from_macos_layer(&self, layer: *mut std::ffi::c_void) -> Surface {
```

Comment:
Is this being removed?

Reply:
indeed, we should keep that one

Reply:
Is it too late to give my feedback on the `raw-window-handle` stuff or is this something decided? It seems very unfinished as an idea, and I do mean just the idea alone sans implementation. https://github.com/rust-windowing/raw-window-handle/issues/14

Reply:
I think we can live with NSView/UIView, but the discussion about `CAMetalLayer` does still need to happen. Thanks for filing the issue!